source/text/Utf8.cpp

```diff
@@ -23,6 +23,14 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 
 using namespace std;
 
+
+
```

Comment:
There's usually only one line of space between the `using` and the `namespace` lines.
```suggestion
```